### PR TITLE
request: implement access to request queue from modal

### PR DESCRIPTION
**Pull request by AoNoOokami**
* Corrects private variables to follow project syntax.
* Adds number of requests on top of modal window if there is a queue.
* Implements access to queue in item detail view from modal.
* Removes weird character in licence text.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

https://tree.taiga.io/project/rero21-reroils/task/1418?kanban-status=1224894
To implement access to request queue in item detail view from 'add request' modal window.

## How to test?

Go to a document detail view with request(s).
Click on 'request' button for an item.
A modal window with a list of pending request (and 'add request' form) should be displayed:
modal

Click on 'Edit queue' button. You should be redirected to the item detailed view.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.html [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (sebastiendeleze) [inline projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.html:15]: I think we can do a specific PR to fix this in the whole project

File: projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.html [hunk 2]
```diff
@@ -24,6 +24,25 @@ <h5 class="modal-title" translate>Item request</h5>
...
+                {{ requestedBy.length }} {{ 'request in the queue' | translate }}
+              </ng-container>
+              <ng-template #pluralBlock>
+                <i class="fa fa-exclamation-triangle pr-1" aria-hidden="true"></i>
+                {{ requestedBy.length }} {{ 'requests in the queue' | translate }}
+              </ng-template>
+            </div>
+            <div class="col-auto">
+              <button type="button" class="btn btn-outline-primary btn-link pt-2 pb-2"
+              [routerLink]="['/records','items','detail', itemPid]"
```
Comment (sebastiendeleze) [inline projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.html:41]: spaces are missing after comma.

File: projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.ts [hunk 3]
```diff
@@ -15,31 +15,33 @@
...
-
   /** Pickup default $ref */
   private pickupDefaultValue: string;
 
   /** Current user */
   private currentUser: any;
 
+  /** Item pid */
+  itemPid: string;
+
```
Comment (sebastiendeleze) [inline projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.ts:44]: public properties must be placed before private ones.

File: projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.ts [hunk 4]
```diff
@@ -75,7 +80,8 @@ export class ItemRequestComponent implements OnInit {
     private _recordService: RecordService,
     private _http: HttpClient,
     private _toastr: ToastrService,
-    private _translateService: TranslateService
+    private _translateService: TranslateService,
+    private _loanService: LoanService
```
Comment (sebastiendeleze) [inline projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.ts:84]: Missing @param for this argument.

File: projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.html (hunk 1 above)
Comment (AoNoOokami) [reply projects/admin/src/app/record/detail-view/document-detail-view/item-request/item-request.component.html:15]: Yes. But that would be nice to find where those strange characters come from.

**Pull request merged**